crates/invoker-impl/src/metric_definitions.rs

```diff
@@ -18,7 +18,9 @@ pub const INVOKER_INVOCATION_TASKS: &str = "restate.invoker.invocation_tasks.tot
 pub const INVOKER_AVAILABLE_SLOTS: &str = "restate.invoker.available_slots";
 pub const INVOKER_CONCURRENCY_LIMIT: &str = "restate.invoker.concurrency_limit";
 pub const INVOKER_TASK_DURATION: &str = "restate.invoker.task_duration.seconds";
+pub const INVOKER_SERVICE_RESPONSE_TIME: &str = "restate.invoker.service_response_time.seconds";
```

Comment:
1- s/_time/_duration/ to maintain consistency and convention
2- I think you mean the `deployment` not the `service`.
3- Is this the time-to-first-byte from the deployment? should we name it accordingly?